### DISCUSSION TITLE
Any option to specify which event triggers a request ?

### DISCUSSION THREAD

**Discussion by naveencom** [unpoly/unpoly#485] [Q&A] [answered]
[https://www.w3schools.com/tags/ref_eventattributes.asp](https://www.w3schools.com/tags/ref_eventattributes.asp)

Is there an attribute to specify which event will trigger a reuqest ? 

Anything similar like hx-trigger of HTMX ?

[https://htmx.org/attributes/hx-trigger/](https://htmx.org/attributes/hx-trigger)

**Comment by adam12** [comment, marked as answer]
There's no equivalent to `hx-trigger`. Unpoly will ignore non-plain clicks (alt-click, etc) by default.

You could probably simulate `hx-trigger` with your own Unpoly compiler, an event listener, and `up.request`.

**Reply by naveencom** [reply]
@adam12 @triskweline Please consider adding something similar in future version. Thanks.
